### PR TITLE
doc: fix minioJob env_ref_doc

### PR DESCRIPTION
**Pull request by jiuker**
doc: fix minioJob env_ref_doc
fix https://github.com/minio/operator/issues/2233

### CODE REVIEW

**Review by shtripat — APPROVED** [review]
lgtm

**Review by pjuarezd — APPROVED** [review]

File: pkg/controller/job-controller.go [hunk 1]
```diff
@@ -153,6 +153,7 @@ func NewJobController(
 			}
 			controller.HandleObject(newJob)
 		},
+		DeleteFunc: controller.enqueueJob,
```
Comment (ramondeklein) [inline pkg/controller/job-controller.go:156]: Is this related to this PR?

File: docs/minio-job.md [hunk 2]
```diff
@@ -78,16 +78,16 @@ spec:
     - name: add-my-user-1
       op: admin/user/add
       args:
-        user: ${USER}
-        password: ${PASSWORD}
+        user: $(USER)
```
Comment (ravindk89) [inline docs/minio-job.md:81]: `${VAR}` is a bash-ism right?

`$(VAR)` is a bit atypical, or in bash semantics would imply a subshell.

Is this the easiest path, or would changing the logic to support `${VAR}` substitution be harder?

Just curious.

**Review by ravindk89 — APPROVED** [review]
